### DISCUSSION TITLE
Title and Comment changes in MSI installer

### DISCUSSION THREAD

**Discussion by BartoszGoralski** [wixtoolset/issues#7647] [Questions]
Hello all!

I'm a newbie at Wix Toolset. We try to migrate from InstallShield, and in V4.0.1 Wix Toolset i cannot change Comment and Title sections.
In v.3 WIX there was an easy way to use "Package Element" and on the web there are a lot of examples. In V4 when I declare sections  or there is no possibility to change any of those sections for installer.

Any suggestions? I have seen that the "Comment" section declarated by user will be (perhabs) available in the next release (after checking a source code, and some merges of code).

**Comment by robmen** [comment]
Why do you want to be able to do that? What are you setting those values to?

**Reply by BartoszGoralski** [reply]
Internal company assumptions.
Title - was information about name of installed product and his visible version (f. ex. Repair Pack - Tools 1.32.1)
Comment - an extra information about installer.
